app/pb_kits/playbook/pb_source/docs/_description.md

```diff
@@ -0,0 +1 @@
+Source describes how a customer discovered PHRG.
```

Comment:
Remove PHRG and rephrase to cover photo please industries.

Reply:
@jasoncypret Updated source kit screenshot

Reply:
I don't see that this change was push up